workday/15.0/modules/ROOT/pages/workday-reference.adoc

```diff
@@ -250,28 +267,28 @@ a|  The Workday object type that applies to this input source. The following obj
 * <<workdayExtensibility>>
 * <<workforcePlanning>>
 
-== Operations
 
 [[absenceManagement]]
 === Absence Management
 
 `<workday:absence-management>`
 
-Invoke the Absence Management web service. The Absence Management web service contains operations that expose absence-related data, including employee time off and absence inputs for time off and accrual adjustments and overrides, and leave requests, in Workday Human Capital Management business services.
+Invokes the Absence Management web service. The Absence Management web service contains operations that expose absence-related data, including employee time off and absence inputs for time off and accrual adjustments and overrides, and leave requests, in Workday Human Capital Management business services.
```

Comment:
The second sentence is large, so it's hard to follow. I would split it in two. Maybe:

The Absence Management web service contains operations that expose absence-related data in Workday Human Capital Management  business service.  This data includes employee time off, absence inputs for time off, accrual adjustments and overrides for time off, and leave requests.

Reply:
much better!